Solving Issue #3055 (#3414)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md

- [Windows Defender Advanced Threat Protection (Windows Defender ATP)](https://go.microsoft.com/fwlink/p/?linkid=2069559)

> >[!IMPORTANT]

Reviewer: ```suggestion
> [!IMPORTANT]
```

> > [Windows Defender ATP](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-atp/defender-compatibility-windows-defender-advanced-threat-protection) does not adhere to the Windows Defender Antivirus Exclusions settings. In other words, Windows Defender Exclusions,however they are created, are not used by ATP.

Reviewer: ```suggestion
> [Windows Defender Advanced Threat Protection ](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-atp/defender-compatibility-windows-defender-advanced-threat-protection) does not adhere to Windows Defender Antivirus exclusion settings. This means that any Windows Defender exclusions, no matter how you created them, are not applied by Windows Defender ATP.
```

> 

Reviewer 2: 1 blank line after the quoted **&#9432; important** section is enough. The existing **&#9432; important** sections use only 1 blank line after.

> 

Reviewer: ```suggestion
```

>[!IMPORTANT]

> >The use of wildcards such as the asterisk (\*) will alter how the exclusion rules are interpreted. See the [Use wildcards in the file name and folder path or extension exclusion lists](#use-wildcards-in-the-file-name-and-folder-path-or-extension-exclusion-lists) section for important information about how wildcards work.

Reviewer 2: Unfortunately, "Applying suggestions on deleted lines is not supported."
Please add back this missing line.

> >You cannot exclude mapped network drives. You must specify the actual network path.

Reviewer 2: Please add back this missing line.

>[!IMPORTANT]
>The use of wildcards such as the asterisk (\*) will alter how the exclusion rules are interpreted. See the [Use wildcards in the file name and folder path or extension exclusion lists](#use-wildcards-in-the-file-name-and-folder-path-or-extension-exclusion-lists) section for important information about how wildcards work.

> >You cannot exclude mapped network drives. You must specify the actual network path.

Reviewer 2: Judging by the looks of the **&#9432; important** sections in https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus, I don't think it is practical to remove the blank lines from these sections. The blank lines are there for readability. (see also the next **&#9432; important** section)

>[!IMPORTANT]

> >Exclusion list changes made with Group Policy **will show** in the lists in the [Windows Security app](windows-defender-security-center-antivirus.md#exclusions).

Reviewer 2: Please add back this missing line.

>[!IMPORTANT]
>Exclusion list changes made with Group Policy **will show** in the lists in the [Windows Security app](windows-defender-security-center-antivirus.md#exclusions).
>Changes made in the Windows Security app **will not show** in the Group Policy lists.

By default, local changes made to the lists (by users with administrator privileges, including changes made with PowerShell and WMI) will be merged with the lists as defined (and deployed) by Group Policy, Configuration Manager, or Intune. The Group Policy lists will take precedence in case of conflicts.

You can [configure how locally and globally defined exclusions lists are merged](configure-local-policy-overrides-windows-defender-antivirus.md#merge-lists) to allow local changes to override managed deployment settings.

> 

Reviewer 2: This blank line is redundant. I don't think it adds anything useful with regards to formatting (it also extends the length of the affected source needlessly).

> 

Reviewer: ```suggestion
```
